src/main/java/frc/robot/CommandSwerveDrivetrain.java

```diff
@@ -19,6 +25,7 @@ public class CommandSwerveDrivetrain extends SwerveDrivetrain implements Subsyst
   private static final double kSimLoopPeriod = 0.005; // 5 ms
   private Notifier m_simNotifier = null;
   private double m_lastSimTime;
+  private SwerveDriveKinematics kinematics = new SwerveDriveKinematics();//TODO need translations
```

Comment:
Should be able to build the Translation2ds from the SwerveModuleConstants arguments. LocationX and LocationY are public readable constants in each that class.